userguide/plugins.rst

```diff
@@ -389,40 +389,93 @@ describes each required entry.
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"plugin_schema":`  | Optional. Enter *2* if simplified post install information has     |
+   |                           | been supplied in :file:`post_install.sh`. After specifying *2*,    |
+   |                           | echo the information to be presented to the user in                |
+   |                           | :file:`/root/PLUGIN_INFO` inside the                               |
+   |                           | :file:`post_install.sh` file.                                      |
+   |                           | See the :ref:`rslsync.json <rslsync-plugin-schema>` and            |
+   |                           | :ref:`rslsync post_install.sh <rslsync-post-install>` examples.    |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"release":`        | FreeBSD RELEASE to use for the plugin jail.                        |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"artifact":`       | URL of the plugin artifact repo.                                   |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"pkgs":`           | Port of the plugin.                                                |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"packagesite":`    | CDN the plugin jail uses. Default for the TrueOS CDN is            |
+   |                           | http://pkg.cdn.trueos.org/iocage .                                 |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"fingerprints":`   | :samp:`"function":`                                                |
+   |                           |                                                                    |
+   |                           | Default is *sha256*.                                               |
+   |                           |                                                                    |
+   |                           | :samp:`"fingerprint":`                                             |
```

Comment:
As above, when these are the literal strings the user is meant to enter, use :literal:.